src/main/java/me/damonkelley/ui/GamePresenter.java

```diff
@@ -2,63 +2,90 @@
 
 import me.damonkelley.tictactoe.Game;
 import me.damonkelley.tictactoe.Marker;
-import me.damonkelley.tictactoe.Space;
+
+import java.util.ArrayList;
 
 class GamePresenter {
-    private StringBuffer output = new StringBuffer();
     private Game game;
     private int id = 1;
 
+    private final String THREE_BY_THREE = " %s | %s | %s \n" +
+                                          "---+---+---\n" +
+                                          " %s | %s | %s \n" +
+                                          "---+---+---\n" +
+                                          " %s | %s | %s \n" +
+                                          "\n";
+
+   private final String FOUR_BY_FOUR =    " %s | %s | %s | %s \n" +
+                                          "-----+-----+-----+-----\n" +
+                                          " %s | %s | %s | %s \n"  +
+                                          "-----+-----+-----+-----\n" +
+                                          " %s | %s | %s | %s \n"  +
+                                          "-----+-----+-----+-----\n" +
+                                          " %s | %s | %s | %s \n"  +
+                                          "\n";
+
     public GamePresenter(Game game) {
         this.game = game;
     }
 
     public String present() {
-        for (Space space : game.getBoard()) {
-            addSpace(space);
-            addSeparator();
+        String template = (game.getBoard().getSize() == 4)? FOUR_BY_FOUR : THREE_BY_THREE;
```

Comment:
It's a little funky that you're making this decision every time you present the board - seems like something that you could just do once, perhaps in the constructor?